packages/mobile/src/exchange/FeeExchangeEducation.tsx

```diff
@@ -0,0 +1,60 @@
+import CircleButton from '@celo/react-components/components/CircleButton'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import * as React from 'react'
+import { withNamespaces, WithNamespaces } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import componentWithAnalytics from 'src/analytics/wrapper'
+import { ringsPlusCircle } from 'src/images/Images'
+import { navigateBack } from 'src/navigator/NavigationService'
+
+type Props = WithNamespaces
+
+class FeeExchangeEducation extends React.PureComponent<Props> {
+  static navigationOptions = { header: null }
+
+  render() {
+    const { t } = this.props
+    return (
+      <View style={styles.container}>
+        <Image source={ringsPlusCircle} resizeMode={'contain'} style={styles.image} />
+        <View style={styles.textBox}>
+          <Text style={[fontStyles.telephoneHeadline, styles.text]}>
+            {t('feeExchangeEducation')}
+          </Text>
+        </View>
+        <View style={styles.buttonContainer}>
+          <CircleButton onPress={navigateBack} solid={true} disabled={false} />
+        </View>
+      </View>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.background,
+    flex: 1,
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+  },
+  image: {
+    width: 240,
+    height: 220,
+  },
+  textBox: {
+    marginVertical: 30,
+    paddingHorizontal: 50,
+    alignItems: 'center',
+  },
+  text: {
+    textAlign: 'center',
+    fontSize: 24,
+    lineHeight: 30,
+  },
+  buttonContainer: {
+    marginVertical: 40,
+  },
+})
+
+export default componentWithAnalytics(withNamespaces('exchangeFlow9')(FeeExchangeEducation))
```

Comment:
Nit: use the enum value (Namespaces.FOO) instead of the hard coded string for exchangeFlow9 here

Reply:
> Mostly looks good! In addition to the comments please:
> 
> 1. Create a new issue for retrieving the tobin tax
> 2. Add screenshots to the PR (we typically do this for UI changes)
> 3. Add a snapshot test for the new FeeExchangeEducation screen

Can you provide me more information about snapshot tests?